src/main/webapp/college-info.js

```diff
@@ -289,8 +289,61 @@ function createCommentElement(name, message, time) {
 function prepReviewForm(id) {
   const submitReviewForm = document.getElementById('submit-review');
   submitReviewForm.setAttribute('action', `/data?id=${id}`);
+
   const idInputElement = document.getElementById('school-id');
   idInputElement.setAttribute('value', id);
 }
 
+/**
+ * Sets the hidden elements to be shown and the shown elements to be hidden.
+ * @param {HTMLElement} showElement HTML element that is hidden/will be shown.
+ * @param {HTMLElement} hideElement HTML element that is shown/will be hidden.
+ * @param {HTMLElement} showElementNavButton HTML nav button to activate.
+ * @param {HTMLElement} hideElementNavButton HTML nav button to deactivate.
+ */
+function toggleElementsDisplay(showElement, hideElement, showElementNavButton,
+    hideElementNavButton) {
+  showElement.classList.remove('is-hidden');
+  hideElement.classList.add('is-hidden');
+
+  showElementNavButton.classList.add('active');
+  hideElementNavButton.classList.remove('active');
+}
+
+/**
+ * Sets the toggle functions to the nav bar buttons.
+ */
+function loadToggle() {
+  const infoDiv = document.getElementById('info');
+  const reviewsDiv = document.getElementById('reviews');
+  const infoNavButton = document.getElementById('info-nav');
+  const reviewsNavButton = document.getElementById('reviews-nav');
+
+  // Gets location hash from URL to show/hide appropriate section when the
+  // page initially loads.
+  if (location.hash == '#reviews') {
+    infoDiv.classList = 'is-hidden';
```

Comment:
I don't think it's really intended to set `classList`. If you want to set the full class string on the element, use infoDiv.className = '...' but usually it's probably better for you to just classList.add() this class. That way you don't break things if someone tries to add other classes to the element in the future.